frontend/src/features/admin-form/settings/components/AuthSettingsSection/AuthSettingsSectionContainer.tsx

```diff
@@ -1,9 +1,7 @@
 import { useAdminFormSettings } from '../../queries'
 
-import {
-  AuthSettingsSection,
-  AuthSettingsSectionSkeleton,
-} from './AuthSettingsSection'
+import { AuthSettingsSection } from './AuthSettingsSection'
+import { AuthSettingsSectionSkeleton } from './AuthSettingsSectionSkeleton'
```

Comment:
extracted to individual file in b0b606e5e

Reply:
Rationale: prefer only to have one **exported** React Component per file and that exported file should match component name.

Files can have several other un-exported React Components

Reply:
got it! i will keep this in mind in the future!